### ISSUE TITLE
validation sample: bad quality images

### ISSUE THREAD

**Issue by kolar1988** [bghira/SimpleTuner#330]
I trained sdxl model, but got bad image. 

![desktop](https://github.com/bghira/SimpleTuner/assets/39948604/b232e3e6-b6b5-45fd-a937-e02852eea45e)

```
SimpleTuner/train_sdxl.py --model_type=full --pretrained_model_name_or_path=stabilityai/stable-diffusion-xl-base-1.0 \
--enable_xformers_memory_efficient_attention --gradient_checkpointing --set_grads_to_none --gradient_accumulation_steps=8 \
--resume_from_checkpoint=latest --snr_gamma=5 --data_backend_config=./multidatabackend.json --num_train_epochs=30 \
--max_train_steps=1000 --metadata_update_interval=65 --use_8bit_adam --learning_rate=8e-7 --lr_scheduler=constant \
--seed 420420420 --lr_warmup_steps=0 --output_dir=./outputs --inference_scheduler_timestep_spacing=leading \
--training_scheduler_timestep_spacing=leading --report_to=wandb --allow_tf32 --mixed_precision=bf16 --vae_dtype=bf16 \
--pretrained_vae_model_name_or_path ./sdxl-vae-fp16-fix --override_dataset_config --train_batch=10 \
--caption_dropout_probability=0.1 --validation_prompt="a woman riding a bicycle on a city street" --num_validation_images=1 \
--validation_num_inference_steps=20 --validation_seed=42 --minimum_image_size=1024 --resolution=1024 \
--validation_resolution=1024 --resolution_type=pixel --checkpointing_steps=50 --checkpoints_total_limit=100 \
--validation_steps=10 --tracker_run_name=simpletuner-sdxl --tracker_project_name=sdxl-training \
--validation_guidance=7.5 --validation_guidance_rescale=0.0 --validation_negative_prompt="blurry, cropped, ugly"
```

Learning rate I've tried a few and they don't work: 1e-5, 8e-6, 8e-7.

multidatabackend.json: 
```
[
    {
        "id": "street_datasets",
        "type": "local",
        "instance_data_dir": "./img",
        "crop": false,
        "crop_style": "random",
        "crop_aspect": "preserve",
        "resolution": 1024,
        "resolution_type": "pixel",
        "minimum_image_size": 1.0,
        "prepend_instance_prompt": false,
        "instance_prompt": "",
        "only_instance_prompt": false,
        "caption_strategy": "textfile",
        "cache_dir_vae": "./cache_vae",
        "vae_cache_clear_each_epoch": false,
        "probability": 1.0,
        "repeats": 1,
        "text_embeds": "alt-embed-cache"
    },
    {
        "id": "alt-embed-cache",
        "dataset_type": "text_embeds",
        "default": true,
        "type": "local",
        "cache_dir": "./textembed_cache"
    }
]
```

Most of the original train images have more than a million pixels, e.g. [3596, 5394], [6720, 4480].
```
(Rank: 0)  | Bucket     | Image Count
------------------------------
(Rank: 0)  | 0.667      | 3040
(Rank: 0)  | 1.508      | 0
(Rank: 0)  | 1.5        | 2080
(Rank: 0)  | 1.773      | 0
(Rank: 0)  | 0.663      | 0
(Rank: 0)  | 0.8        | 160
(Rank: 0)  | 1.336      | 0
(Rank: 0)  | 0.749      | 80
```

**Comment by bghira** [comment]
it'll need more information than that. as it stands, i can't see anything wrong here. it could be your training data, it could be the resolutions you're training at. resolution_type=area and resolution=1 might work better, who knows. no idea how many steps you trained for, etc

**Comment by zhuliyi0** [comment]
Try some different learning rate, start with the biggest batch size you can use to quickly iterate. I suspect 8e-7 is a bit small. Batch size 10-20 and a learning rate of 1e-5 worked best for my dataset.

**Comment by kolar1988** [comment]
> it'll need more information than that. as it stands, i can't see anything wrong here. it could be your training data, it could be the resolutions you're training at. resolution_type=area and resolution=1 might work better, who knows. no idea how many steps you trained for, etc它需要更多的信息。就目前情况而言，我看不出有什么问题。它可能是您的训练数据，也可能是您正在训练的分辨率。分辨率类型=区域和分辨率=1可能效果更好，谁知道呢。不知道您训练了多少步等

Ok, thanks. Now I'm trying other ways to fix it

**Comment by kolar1988** [comment]
> Try some different learning rate, start with the biggest batch size you can use to quickly iterate. I suspect 8e-7 is a bit small. Batch size 10-20 and a learning rate of 1e-5 worked best for my dataset.尝试一些不同的学习率，从可用于快速迭代的最大批量大小开始。我怀疑8e-7有点小。批量大小 10-20 和学习率 1e-5 最适合我的数据集。

Learning rate I had tried to turn up to 5e-5 but it still didn't work well

**Comment by zhuliyi0** [comment]
make sure the data is good, including the image and the caption. then try various lr see what you get.

if still not ideal, try limiting your data to one or two specify styles /types / genres to reduce the learning difficulty, and see if it can learn it better. I personally find SDXL is very quick to learn pattern, but may struggle to learn a "sparse" dataset with many unrelated concepts in it.
